--- kde-i18n-3.2.1/nl/docs/kdeedu/ktouch/index.docbook.org	2004-03-15 11:58:36.000000000 +0000
+++ kde-i18n-3.2.1/nl/docs/kdeedu/ktouch/index.docbook	2004-03-15 11:59:13.000000000 +0000
@@ -29,7 +29,6 @@
 </affiliation>
 </author>
 
-&Tijmen.Baarda;&Rinse.Devries; 
 
 </authorgroup>
 
@@ -465,8 +464,7 @@
 ></listitem>
 </itemizedlist>
 
-&meld.fouten;&vertaling.tijmen;&nagelezen.rinse; 
-&underFDL; &underGPL; </chapter>
+</chapter>
 
 <appendix id="installation">
 <title
