--- kde-i18n.orig/admin/cvs.sh.orig	2004-03-14 23:14:22.000000000 +0000
+++ kde-i18n/admin/cvs.sh	2004-03-14 23:19:48.000000000 +0000
@@ -233,10 +233,10 @@
 if test -f Makefile.am.in; then
   subdirs=`cat subdirs`
   for dir in $subdirs; do
-    dir=`echo $dir | sed -e "s,[-+.],_,g"`
-    echo "AM_CONDITIONAL($dir""_SUBDIR_included, test \"x\$$dir""_SUBDIR_included\" = xyes)" >> configure.in.new
-    if test -f $dir/configure.in; then
-	echo "if test \"x\$$dir""_SUBDIR_included\" = xyes; then " >> configure.in.new
+    vdir=`echo $dir | sed -e "s,[@-+.],_,g"`
+    echo "AM_CONDITIONAL($vdir""_SUBDIR_included, test \"x\$$vdir""_SUBDIR_included\" = xyes)" >> configure.in.new
+    if test -f "$dir/configure.in"; then
+	echo "if test \"x\$$vdir""_SUBDIR_included\" = xyes; then " >> configure.in.new
 	echo "  AC_CONFIG_SUBDIRS($dir)" >> configure.in.new
 	echo "fi" >> configure.in.new
     fi
