diff -urN kde-i18n-2.2.1.org/ko/Makefile.am kde-i18n-2.2.1/ko/Makefile.am
--- kde-i18n-2.2.1.org/ko/Makefile.am	Thu Oct 18 20:00:29 2001
+++ kde-i18n-2.2.1/ko/Makefile.am	Thu Oct 18 20:00:57 2001
@@ -1,5 +1,5 @@
 #created by toplevel Makefile
-SUBDIRS = docs messages
+SUBDIRS = docs 
 AUTOMAKE_OPTIONS = foreign
 EXTRA_DIST = configure.in.in
 $(srcdir)/configure.in.in: $(srcdir)/../Makefile.am
diff -urN kde-i18n-2.2.1.org/ko/Makefile.in kde-i18n-2.2.1/ko/Makefile.in
--- kde-i18n-2.2.1.org/ko/Makefile.in	Thu Oct 18 20:00:29 2001
+++ kde-i18n-2.2.1/ko/Makefile.in	Thu Oct 18 20:00:49 2001
@@ -212,7 +212,7 @@
 x_includes = @x_includes@
 x_libraries = @x_libraries@
 
-SUBDIRS = docs messages
+SUBDIRS = docs 
 AUTOMAKE_OPTIONS = foreign
 EXTRA_DIST = configure.in.in
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
diff -urN kde-i18n-2.2.1.org/pl/messages/kdebase/kio_smbro.po kde-i18n-2.2.1/pl/messages/kdebase/kio_smbro.po
--- kde-i18n-2.2.1.org/pl/messages/kdebase/kio_smbro.po	Thu Oct 18 20:00:29 2001
+++ kde-i18n-2.2.1/pl/messages/kdebase/kio_smbro.po	Thu Oct 18 20:04:15 2001
@@ -71,14 +71,6 @@
 "Patrz KDE Centrum Sterowania w sekcji SieÄ, PrzeglÄdanie sieci lokalnej aby "
 "uzyskaÄ wiÄcej informacji."
 
-#: kio_smb.cpp:937 kio_smb.cpp:968
-msgid ""
-"\n"
-"Could not create required pipe %1 ."
-msgstr ""
-"\n"
-"Nie moÅ¼na utworzyÄ wymaganego potoku %1."
-
 #: kio_smb.cpp:979 kio_smb.cpp:987
 #, fuzzy
 msgid ""
diff -urN kde-i18n-2.2.1.org/th/Makefile.am kde-i18n-2.2.1/th/Makefile.am
--- kde-i18n-2.2.1.org/th/Makefile.am	Thu Sep 20 06:53:43 2001
+++ kde-i18n-2.2.1/th/Makefile.am	Thu Oct 18 20:18:01 2001
@@ -1,5 +1,5 @@
 #created by toplevel Makefile
-SUBDIRS = messages
+SUBDIRS = 
 AUTOMAKE_OPTIONS = foreign
 EXTRA_DIST = configure.in.in
 $(srcdir)/configure.in.in: $(srcdir)/../Makefile.am
diff -urN kde-i18n-2.2.1.org/th/Makefile.in kde-i18n-2.2.1/th/Makefile.in
--- kde-i18n-2.2.1.org/th/Makefile.in	Thu Sep 20 06:53:47 2001
+++ kde-i18n-2.2.1/th/Makefile.in	Thu Oct 18 20:17:53 2001
@@ -212,7 +212,7 @@
 x_includes = @x_includes@
 x_libraries = @x_libraries@
 
-SUBDIRS = messages
+SUBDIRS = 
 AUTOMAKE_OPTIONS = foreign
 EXTRA_DIST = configure.in.in
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
diff -urN kde-i18n-2.2.1.org/zh_TW.Big5/Makefile.am kde-i18n-2.2.1/zh_TW.Big5/Makefile.am
--- kde-i18n-2.2.1.org/zh_TW.Big5/Makefile.am	Thu Sep 20 06:54:07 2001
+++ kde-i18n-2.2.1/zh_TW.Big5/Makefile.am	Thu Oct 18 20:21:31 2001
@@ -1,5 +1,5 @@
 #created by toplevel Makefile
-SUBDIRS = docs messages
+SUBDIRS = docs 
 AUTOMAKE_OPTIONS = foreign
 EXTRA_DIST = configure.in.in
 $(srcdir)/configure.in.in: $(srcdir)/../Makefile.am
diff -urN kde-i18n-2.2.1.org/zh_TW.Big5/Makefile.in kde-i18n-2.2.1/zh_TW.Big5/Makefile.in
--- kde-i18n-2.2.1.org/zh_TW.Big5/Makefile.in	Thu Sep 20 06:54:14 2001
+++ kde-i18n-2.2.1/zh_TW.Big5/Makefile.in	Thu Oct 18 20:21:41 2001
@@ -212,7 +212,7 @@
 x_includes = @x_includes@
 x_libraries = @x_libraries@
 
-SUBDIRS = docs messages
+SUBDIRS = docs 
 AUTOMAKE_OPTIONS = foreign
 EXTRA_DIST = configure.in.in
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
