--- kde-i18n-3.2.1/nl/docs/kdeedu/ktouch/index.docbook.org	2004-03-15 11:58:36.000000000 +0000
+++ kde-i18n-3.2.1/nl/docs/kdeedu/ktouch/index.docbook	2004-03-15 11:59:13.000000000 +0000
@@ -29,7 +29,6 @@
 </affiliation>
 </author>
 
-&Tijmen.Baarda;&Rinse.Devries; 
 
 </authorgroup>
 
@@ -465,8 +464,7 @@
 ></listitem>
 </itemizedlist>
 
-&meld.fouten;&vertaling.tijmen;&nagelezen.rinse; 
-&underFDL; &underGPL; </chapter>
+</chapter>
 
 <appendix id="installation">
 <title
--- kde-i18n-3.2.2/nl/docs/kdepim/karm/index.docbook.org	2004-04-20 13:34:52.000000000 +0000
+++ kde-i18n-3.2.2/nl/docs/kdepim/karm/index.docbook	2004-04-20 13:35:15.000000000 +0000
@@ -339,7 +339,7 @@
 <menuchoice
 ><shortcut
 ><keycap
->&Esc;</keycap
+></keycap
 ></shortcut
 > <guimenu
 >Clock</guimenu
--- kde-i18n-3.2.2/nl/docs/kdeedu/kmplot/using.docbook.org	2004-04-20 14:01:10.000000000 +0000
+++ kde-i18n-3.2.2/nl/docs/kdeedu/kmplot/using.docbook	2004-04-20 14:02:10.000000000 +0000
@@ -78,7 +78,7 @@
 >Bijvoorbeeld, stel dat u een cirkel wilt tekenen, die de parametrische vergelijkingen heeft x=sin(t), y=cos(t). In de functiesdialoog van &kmplot; doet u het volgende: <orderedlist>
 <listitem
 ><para
->Voer de functie in voor de x-coördinaat, en druk dan op &Enter;: <screen
+>Voer de functie in voor de x-coördinaat, en druk dan op ;: <screen
 ><userinput>
 xcirkel=sin(t)
 </userinput
