--- kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/catman.docbook.org	2004-04-20 10:32:15.000000000 +0000
+++ kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/catman.docbook	2004-04-20 10:32:22.000000000 +0000
@@ -20,7 +20,6 @@
 </author>
 </authorgroup>
 
-&traducteurYvesDessertine; &traducteurPatrickMichelon; 
 </chapterinfo>
 
 <title
--- kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/faq.docbook.org	2004-04-20 12:24:02.000000000 +0000
+++ kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/faq.docbook	2004-04-20 12:24:14.000000000 +0000
@@ -19,7 +19,6 @@
 </author>
 </authorgroup>
 
-&traducteurPatrickMichelon; 
 </chapterinfo>
 
 <title
@@ -89,4 +88,4 @@
 End:
 
 vim:tabstop=2:shiftwidth=2:expandtab 
--->
\ No newline at end of file
+-->
--- kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/glossary.docbook.org	2004-04-20 12:24:26.000000000 +0000
+++ kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/glossary.docbook	2004-04-20 12:24:32.000000000 +0000
@@ -19,7 +19,6 @@
 </author>
 </authorgroup>
 
-&traducteurPatrickMichelon; 
 </glossaryinfo>
 
 <title
@@ -253,4 +252,4 @@
 End:
 
 vim:tabstop=2:shiftwidth=2:expandtab 
--->
\ No newline at end of file
+-->
--- kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/index.docbook.org	2004-04-20 10:30:34.000000000 +0000
+++ kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/index.docbook	2004-04-20 10:31:58.000000000 +0000
@@ -26,7 +26,6 @@
 >&Stanislav.Visnovsky; &Stanislav.Visnovsky.mail; </author>
 <author
 >&Matthias.Kiefer; </author>
-&traducteurPatrickMichelon; 
 </authorgroup>
 
 
--- kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/kbabeldict.docbook.org	2004-04-20 12:22:13.000000000 +0000
+++ kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/kbabeldict.docbook	2004-04-20 12:22:23.000000000 +0000
@@ -19,7 +19,6 @@
 </author>
 </authorgroup>
 
-&traducteurPatrickMichelon; 
 </chapterinfo>
 
 <title
@@ -84,4 +83,4 @@
 End:
 
 vim:tabstop=2:shiftwidth=2:expandtab 
--->
\ No newline at end of file
+-->
--- kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/menu.docbook.org	2004-04-20 12:22:34.000000000 +0000
+++ kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/menu.docbook	2004-04-20 12:22:53.000000000 +0000
@@ -20,7 +20,6 @@
 </author>
 </authorgroup>
 
-&traducteurPatrickMichelon; 
 </chapterinfo>
 
 <title
--- kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/using.docbook.org	2004-04-20 10:31:42.000000000 +0000
+++ kde-i18n-3.2.2/fr/docs/kdesdk/kbabel/using.docbook	2004-04-20 10:32:05.000000000 +0000
@@ -19,7 +19,6 @@
 </author>
 </authorgroup>
 
-&traducteurPatrickMichelon; 
 </chapterinfo>
 
 
@@ -993,4 +992,4 @@
 End:
 
 vim:tabstop=2:shiftwidth=2:expandtab 
--->
\ No newline at end of file
+-->
